### DISCUSSION TITLE
RSSI Problem - RaspberryMatic-3.79.2

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#2853] [🇩🇪 Fragen&Antworten] [answered]
Beim letzten Experimental Nightly Build Snapshots ist ein Fehler beim RSSI-Lesen aufgetreten. 
das gilt für: RaspberryMatic-3.79.2.20240918-b8c2b5-rpi5.zip
Alle Geräte, die ich habe, reagieren gleich.
In der Registerkarte Status and Control/Devices ist ein Fehler aufgetreten aber im Registerkarte Settings/Devices ist alles in Ordnung. 
Ich habe es auf Screenshots gezeigt
![1](https://github.com/user-attachments/assets/23043a1a-85ce-44b0-9a65-9c01c4ae2ec0)
![2](https://github.com/user-attachments/assets/237e9a63-c23b-49e9-8a31-20e2c3bf0e5c)

**Comment by jens-maus** [comment, marked as answer]
Bitte als Bugissue melden.

**Comment by ghost** [comment]
Ich bestätige, dass die neue Version 3.79.2-3 diesen Fehler nicht mehr aufweist.
